Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk:9
+LABEL maintainer="tim.myerscough@mechanicalrock.io"
+
+
+WORKDIR /app
+
+RUN wget http://repo1.maven.org/maven2/io/magentys/donut/1.0/donut-1.0-one-jar.jar /app/donut.jar
```

Comment:
Adding [docker to the Travis build](https://docs.travis-ci.com/user/docker/), caused [test failures](https://travis-ci.org/MagenTys/donut/builds/280777236) due to ordering conflict when reading files.

Therefore building and deploying to DockerHub at the moment would be manual and should use released versions.

Alternatively, we could build the one-jar prior to building the container and copy it in (details are in the README for that)

Reply:
This line could change to 

`COPY target/donut*-one-jar.jar /app/donut.jar`